assembly/src/parsers/nodes.rs

```diff
@@ -269,6 +270,7 @@ impl fmt::Display for Instruction {
             Self::DivImm(value) => write!(f, "div.{value}"),
             Self::Neg => write!(f, "neg"),
             Self::Inv => write!(f, "inv"),
+            Self::Incr => write!(f, "add.1"),
```

Comment:
Likely unrelated to this PR, but this is a bit counter-intuitive to me. Should we add a token `incr` that will be redundant to `add.1`? This way, we will maintain the consistency between the parsed tree and the source file

Reply:
Yeah, this would be a separate issue. We could introduce `incr` instruction at the assembly source level - but regardless of this, `add.1` would need to be translated into `incr` at the AST level.

Let's create an issue to discuss whether it is worth adding `incr` instruction to assembly.